x-pack/legacy/plugins/siem/public/pages/case/components/bulk_actions/index.tsx

```diff
@@ -9,46 +9,51 @@ import { EuiContextMenuItem } from '@elastic/eui';
 import * as i18n from './translations';
 
 interface GetBulkItems {
+  caseStatus: string;
   closePopover: () => void;
   deleteCasesAction: (cases: string[]) => void;
   selectedCaseIds: string[];
-  caseStatus: string;
+  updateCaseStatus: (status: string) => void;
 }
 
 export const getBulkItems = ({
-  deleteCasesAction,
-  closePopover,
   caseStatus,
+  closePopover,
+  deleteCasesAction,
   selectedCaseIds,
+  updateCaseStatus,
 }: GetBulkItems) => {
   return [
     caseStatus === 'open' ? (
       <EuiContextMenuItem
+        data-test-subj="cases-bulk-close-button"
         key={i18n.BULK_ACTION_CLOSE_SELECTED}
         icon="magnet"
-        disabled={true} // TO DO
-        onClick={async () => {
+        onClick={() => {
```

Comment:
Better to use `useCallback` to avoid unnecessary re-renders

Reply:
I'm not in a react component here. FWIW the actions passed here are in their own `useCallback`s

Reply:
I am not sure given that EuiContextMenuItem is a component. @patrykkopycinski ?

Reply:
I think @stephmilovic is right, but to be sure you can always use `why-did-you-render` or Profiler :)